cmd/repo_add_test.go

```diff
@@ -99,7 +99,7 @@ func TestRepoAddErrors(t *testing.T) {
 		{"No args", nil, "", "You must specify a repository name and URL"},
 		{"One arg", []string{"reponame"}, "", "You must specify a repository name and URL"},
 		{"No url scheme", []string{"test", "localhost"}, "", "unsupported protocol scheme"},
-		{"Non-existing url", []string{"test", "http://localhost/doesnotexist"}, "", "refused"},
+		{"Non-existing url", []string{"test", "http://localhost/doesnotexist"}, "", "Error"},
```

Comment:
Is there a better error message that could be returned here?

Reply:
There was an issue with this test @kilnerm brought up a while ago (that I managed to replicate). Depending on whether you are connected to IBM or IBMInternet wifi, we get differing error messages- has to do with the way IBMInternet reroutes unknown URLs I think. We still want the command to fail if the URL is unknown, so I've kept the test in, but we can't predict which error message will be given - only that one will be given

Reply:
Makes sense, maybe we could add a generic error message that wraps the error returned from the download function [here](https://github.com/appsody/appsody/blob/master/cmd/repo_add.go#L75) - something like `Error("Error downloading index: %s", err)`